changelog/unreleased/kong/move_unix_sockets.yml

```diff
@@ -0,0 +1,3 @@
+message: Moved Unix domain sockets to /var/run/kong to prevent issues when mounting the prefix directory in containerized environments
+type: bugfix
```

Comment:
is this the proper scope? considering all things involved, permissions et al, should we put it in the changelog as a change instead of bugfix? even if just for awareness?

Reply:
The changelog scope must be one of these based on the new changelog process: "feature", "bugfix", "dependency", "deprecation", "breaking_change", "performance" -- are you saying to mark it as a breaking_change?